lib/github_storage_policy.rb

```diff
@@ -1,15 +1,16 @@
 # frozen_string_literal: true
 
 class GithubStoragePolicy
-  def initialize(rules)
+  def initialize(arch, rules)
     @rules = rules || {}
+    @arch = arch
   end
```

Comment:
nit: You might select arch rules here, don't change other lines. It's just a suggestion, you can ignore it
```suggestion
  def initialize(arch, rules)
    @rules = rules[arch] || {}
  end
```